de4a-connector/src/main/java/eu/de4a/connector/service/spring/Conf.java

```
@@ -234,8 +246,7 @@ public SSLContext sslContext() {
 					.loadTrustMaterial(new File(trustStore), trustStorePassword.toCharArray()).build();
 		} catch (IOException | NoSuchAlgorithmException | CertificateException | KeyStoreException
 				| KeyManagementException | UnrecoverableKeyException e) {
-			String msg = "Cannot load certificate";
-			LOG.error(msg, e);
+			LOG.error("There was a problem creating sslContext", e);
 			return null;
```

Same here - please throw an Exception